.storybook/config.js

```diff
@@ -1,50 +0,0 @@
-import { configure, addParameters, addDecorator } from '@storybook/react';
-import { withKnobs } from '@storybook/addon-knobs/react';
-import { setIntlConfig, withIntl } from 'storybook-addon-intl';
-import enUS from '../i18n/en-US';
-import deDE from '../i18n/de-DE';
-import frFR from '../i18n/fr-FR';
-import jaJP from '../i18n/ja-JP';
-import zhCN from '../i18n/zh-CN';
-
-import '../scripts/styleguide.setup.js';
-import customTheme from './customTheme';
-import '../src/styles/variables';
-import '../src/styles/base.scss';
-
-addDecorator(withKnobs);
-
-addParameters({
-    options: {
-        theme: customTheme,
-    },
-    docs: {
-        extractComponentDescription: (component, { notes }) => {
-        if (notes) {
-            return typeof notes === 'string' ? notes : notes.markdown || notes.text;
-        }
-        return null;
-        },
-  },
-});
-
-// Enable translations in stories
-const messages = {
```

Comment:
is this still needed? does the new config handle localization?

Reply:
noticed some stories were throwing errors for intl so i add the decorator here and reactIntl file